app/src/test/java/com/example/atheneum/OwnerCollectionTest.java

```diff
@@ -1,4 +1,107 @@
 package com.example.atheneum;
 
+import org.junit.Test;
+import org.junit.Before;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+
 public class OwnerCollectionTest {
+    private User owner;
+    private OwnerCollection ownedList;
```

Comment:
Calling this variable ownedList is a bit confusing since the type isn't a Java List. It might be more clear to call it ownerCollection or something like that.

Reply:
Good point